tasks/task449_app_review_classification.json

```diff
@@ -0,0 +1,5945 @@
+{
+    "Contributors": [
+        "Vijay Kumawat"
+    ],
+    "Source": [
+        "app_reviews (dataset link- https://huggingface.co/datasets/app_reviews)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "instruction_language": "English",
```

Comment:
@swarooprm is this valid field ? instruction_language ?

Reply:
this is fine.  instruction_language is an optional field

Reply:
ok